src/ui/src/pages/app/monitors/Index/components/MonitorForm/TableConfiguration.tsx

```diff
@@ -8,24 +8,26 @@ import {
 const TableConfiguration: React.FC<{setConfiguration: any}> = ({setConfiguration}) => {
   const [table, setTable] = useState('');
   const [timestampField, setTimestampField] = useState('');
-        
-  function updateFormState(setState: any) {
-       setState();
-       setConfiguration({"table": table, "timestamp_field": timestampField})
-  }
 
+  React.useEffect(() => {
```

Comment:
We should keep the formatting consistent here, don't need `React` in front of the `useEffect` function if imported properly

Reply:
Fair enough